train_configs/llama_1b.toml

```diff
@@ -0,0 +1,40 @@
+# TorchTrain Config.toml
+[job]
```

Comment:
Since there's no official 1b model size for both llama 2/3 release, and the toml files are user facing, It would be better if we only add released model sizes.